pandas/core/indexes/multi.py

```diff
@@ -701,6 +701,15 @@ def array(self):
             "'MultiIndex.to_numpy()' to get a NumPy array of tuples."
         )
 
+    @cache_readonly
+    def dtypes(self):
```

Comment:
could add a return type annotation

Reply:
I'm assuming this is what you mean:
```suggestion
    def dtypes(self) -> Series:
```
However, it looks like we don't necessarily always have access to the **Series** type in this module. Only if the TYPE_CHECKING flag is set.

Reply:
yeah so I think we can just do `"Series"` with the quotes